### ISSUE TITLE
With jspm 0.17 beta, 'jspm depcache' introducing incorrect dependency for many modules

### ISSUE THREAD

**Issue by kwesterfeld** [jspm/jspm-cli#1577]
This a mystery for me, but basically if I use jspm depcache with the jspm 0.17.0-beta.9 release, the jspm.browser.json file will contain incorrect references to nodelibs-process:

```
    "npm:angular-tree-repeat@0.0.1/dist/angular-tree-repeat.js": [
      "github:jspm/nodelibs-process@0.2.0-alpha/#:./process.js"
    ],
```

I'm not sure where this global dependency is getting introduced, or why.  Its very mysterious, but I have 60+ references to this library while never importing it anywhere.  

When the browser goes to pull this depencency, it gets a 404 error due to the # in the dependency.

**Comment by guybedford** [comment]
Thanks, fixed in https://github.com/systemjs/builder/commit/da444c9611254e412b059438431053ce13324b2e.

**Comment by guybedford** [comment]
Released in 0.17.0-beta.10.
